### DISCUSSION TITLE
Awesome work + a possible feature

### DISCUSSION THREAD

**Discussion by ishtms** [heroui-inc/heroui#203] [Feedback]
Looks sleek, and really love the colors on it. Wouldn't it be better to offer a premium edition - where with one-time fee users can access the source code of a lot of different designs, pages, sections like a login page, a cart, a hero section? Something like what chakra-ui's pro version offers.

**Comment by jrgarciadev** [comment]
Hey @Ishtmeet-Singh I'm glad you liked NextUI, at the moment we are making the base of NextUI, maybe in the future we can start thinking in a Pro/Premium edition with more features

Thanks for the suggestion 🙌🏼
